### PR TITLE
Update synology.md

### PR DESCRIPTION
**Pull request by cniweb**
Signed-off-by: Christian Häussler <c-n-i@web.de> (github: @cniweb)

### CODE REVIEW

File: installation/synology.md [hunk 1]
```diff
@@ -14,8 +14,16 @@ Download SPK package from [Releases](https://github.com/openhab/openhab-syno-spk
...
 3.  Click on the Manual Install button.
-4.  In the window that appears, click Choose File and select the .spk file for the package you'd like to install. 
+4.  In the window that appears, click Choose File and select the openHAB .spk file for the package you'd like to install. 
 5.  On the confirmation page, if you would like the package to run immediately after installation, make sure the box next to Run after Installation is ticked. Then click Apply to start installation.
+  
+## Configuration
+
+The addons and configuration files are located at:
+```
+/volume1/public/OpenHAB2/conf
```
Comment (kaikreuzer) [inline installation/synology.md]: why are they not in /volume1/public/openHAB2/conf ? Is there something that needs to be changed about the packaging? :-)

Comment (cniweb) [inline installation/synology.md]: I changed it here: https://github.com/openhab/openhab-syno-spk/commit/e6cc2dad66b8ed81e59bdc4d732acd42d42adccf

Comment (kaikreuzer) [inline installation/synology.md]: I meant:
/volume1/public/OpenHAB2/conf
vs
/volume1/public/openHAB2/conf

Comment (cniweb) [inline installation/synology.md]: OK upper o :-)

Comment (kaikreuzer) [inline installation/synology.md]: Right, so I'd suggest changing this to lower case :-)

**Pull request merged**